apps/slackbot/bolt_app.py

```diff
@@ -52,26 +52,36 @@ def hello_command(ack, body):
 bot = app.client.auth_test()
 print(bot)
 
-def contains_verbose(query: str) -> bool:
-    '''looks for -verbose in the question and returns True or False'''
-    return "-verbose" in query.lower()
-
-def contains_verbosex(query: str) -> bool:
-    '''looks for -verbosex in the question and returns True or False'''
-    return "-verbosex" in query.lower()
-
+# Exported the helper functions to utils.py for better readability
+from utils import format_verbose, show_commands_only, contains_verbose, contains_verbosex
+      
 @app.event("app_mention")
 def event_test(client, say, event):
     question = event['text']
     thread_ts = event.get("thread_ts", None) or event["ts"]
     replies = client.conversations_replies(channel=event['channel'], ts=thread_ts)
     previous_messages = replies['messages'][:-1]
 
-    results, verbose_message = get_response(question, previous_messages)
+    results, logger = get_response(question, previous_messages)
     say(results, thread_ts=thread_ts)
 
-    if contains_verbose(question):
-      say(f"#verbose message: \n```{verbose_message}```", thread_ts=thread_ts)
+    global stored_verbose_message  # initialize stored_verbose_message
+    
+    if contains_verbosex(question):
+      stored_verbose_message = format_verbose(logger)
+      say(f"#verbose message: \n```{stored_verbose_message}```", thread_ts=thread_ts)
+      
+    elif contains_verbose(question):
+      stored_verbose_message = show_commands_only(logger)
+      say(f"#verbose message: \n```{stored_verbose_message}```", thread_ts=thread_ts)
+      
+    else:
+      stored_verbose_message = "No verbose message available"
+
+@app.command("/verbose")
```

Comment:
can we maybe kind of add support for post here?

Reply:
@saminegash Actually I might be wrong here. Don't merge this yet... Let me do some more test.